lib/lightspeed/resource.rb

```diff
@@ -12,6 +12,9 @@ class Link; end
   class Resource
     attr_accessor :id, :attributes, :client, :context, :account
 
+    # Serialization support
+    alias :read_attribute_for_serialization :send
```

Comment:
Hi Allaire 

I'm not entirely comfortable with the all-encompassing nature of this. 

Are you able to hang that off the `as_json` method somehow, possibly even something like

```
def read_attribute_for_serialization(method_name)
end
```